Formula/vgmstream.rb

```diff
@@ -38,7 +48,7 @@ class Vgmstream < Formula
 
   def install
     ENV["LIBRARY_PATH"] = HOMEBREW_PREFIX/"lib"
-    system "cmake", "-S", ".", "-B", "build", "-DBUILD_AUDACIOUS:BOOL=OFF", *std_cmake_args
+    system "cmake", "-S", ".", "-B", "build", "-DBUILD_AUDACIOUS:BOOL=OFF", "-DUSE_CELT=OFF", *std_cmake_args
```

Comment:
This will unconditionally disable support for streams using CELT codecs, versions of which were used by FMOD. vgmstream actually vendors 2 CELT versions for this support.